docker/env/gafl_webapp.env.example

```diff
@@ -26,8 +26,7 @@ SALES_API_URL=http://host.docker.internal:4000
 DEBUG=webapp:*
 
 # Tracking
-#ANALYTICS_PRIMARY_PROPERTY=UA-124887477-9
-#ANALYTICS_XGOV_PROPERTY=UA-124887477-10
+#ANALYTICS_PRIMARY_PROPERTY=G-XXXXXXXX
```

Comment:
You've only updated ANALYTICS_PRIMARY_PROPERTY, but you need to add ANALYTICS_PROPERTY_API as well, don't you?